doc/api/projections_api.rst

```diff
@@ -6,16 +6,13 @@
    :members:
    :show-inheritance:
 
-``matplotlib.projections.polar``
-================================
+Built in projections
```

Comment:
Should have a dash ("built-in", see e.g. https://docs.python.org/3/library/functions.html).  I'll defer to you for the usage in the sentence below, though.